ultimatejobweb/templates/base_html.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>UltimateJob</title>
+    <!-- boostrap css framework-->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <link rel="stylesheet" href="cssfile.css">
+	<!-- jquery cdn-->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-ygbV9kiqUc6oa4msXn9868pTtWMgiQaeYH7/t7LECLbyPA2x65Kgf80OJFdroafW" crossorigin="anonymous"></script>
+       <!-- icons-->
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.5.4/dist/umd/popper.min.js" integrity="sha384-q2kxQ16AaE6UbzuKqyBE9/u/KzioAlnx2maXQHiDX9d4/zp8Ok3f+M7DPm+Ib6IU" crossorigin="anonymous"></script>
+        <!-- boostrap js-->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.min.js" integrity="sha384-pQQkAEnwaBkjpqZ8RU1fF1AKtTcHJwFl3pblpTlHXybJjHpMYo79HY3hIi4NKxyj" crossorigin="anonymous"></script>
+
+</head>
+<body>
+<div class="content">
+  <div class="content-inside">
+<!--jumbotron-->
+<div class="container">
+<div class="jumbotron">
+    <h1 class="display-3">UltimateJob</h1>
+    <p class="lead">This is the place where you will find your perfect career.</p>
+    <hr class="my-4">
+</div>
+</div>
+
+<!--pagination-->
+<div class="container">
+    <nav aria-label="Page navigation example">
+        <ul class="pagination">
+            <li class="page-item">
+                <a href=" / " class="page-link">Home page</a>
+            </li>
+            <li class="page-item">
+                <a href="/signup" class="page-link">Sign up</a>
+            </li>
+            <li class="page-item">
+                <a href="/sign_in" class="page-link">Sign in</a>
+            </li>
+            <li class="page-item">
+                <a href="/personal_area" class="page-link">personal area</a>
+            </li>
+            <li class="page-item">
+                <a href=" /available_jobs" class="page-link">available jobs for you</a>
+            </li>
+        </ul>
+	
+	
+	
+	{% block content %}
+	
+	{% endblock %}
```

Comment:
You want to finish the html file here.
You would probably put the footer and then close the body and html tags.

In the other files should only be the stuff that goes inbetween the block content and endblock tags